server/views/pages/overview/partials/workAndInterestsTabContents.njk

```diff
@@ -9,25 +11,197 @@
+                </a>
+              </dd>
+            </div>
+            {% if workAndInterests.data.workExperience.hasWorkedPreviously | formatYesNo == 'Yes' %}
+            <div class="govuk-summary-list__row">
+              <dt class="govuk-summary-list__key">
+                Type of work experience
+              </dt>
+              <dd class="govuk-summary-list__value">
+                <ul class="govuk-list">
+                  {% for typeOfWorkExperience in workAndInterests.data.workExperience.jobs %}
+                  {% if typeOfWorkExperience.type == 'OTHER' %}
+                    <li>{{ typeOfWorkExperience.other }}</li>
+                  {% else %}
+                    <li>{{ typeOfWorkExperience.type | formatJobType }}</li>
+                  {% endif %}
+                  {% endfor %}
+                </ul>
+              </dd>
+              <dd class="govuk-summary-list__actions">
+                <a class="govuk-link" href="{{ ciagInductionUrl }}/plan/create/{{ prisonerSummary.prisonNumber }}/type-of-work-experience/update">
+                  Change<span class="govuk-visually-hidden"> type of work experience</span>
+                </a>
+              </dd>
+            </div>
+            {% if workAndInterests.data.workExperience.jobs %}
+              {% for workExperienceDetails in workAndInterests.data.workExperience.jobs %}
+                <div class="govuk-summary-list__row">
+                  <dt class="govuk-summary-list__key">
+                    {% if workExperienceDetails.type == 'Other' %}{{ workExperienceDetails.other }}{% else %}{{ workExperienceDetails.type | formatJobType }}{% endif %} experience details
+                  </dt>
+                  <dd class="govuk-summary-list__value">
+                    <p>Job role:<br> {{ workExperienceDetails.role }}</p>
+                    <p>Roles and responsibilities:<br> {{ workExperienceDetails.responsibilities }}</p>
+                  </dd>
+                  <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link" href="{{ ciagInductionUrl }}/plan/create/{{ prisonerSummary.prisonNumber }}/work-details/{% if workExperienceDetails.type == 'Other' %}{{ workExperienceDetails.other | lower }}{% else %}{{ workExperienceDetails.type | lower }}{% endif %}/update">
+                      Change<span class="govuk-visually-hidden"> {% if workExperienceDetails.type == 'Other' %}{{ workExperienceDetails.other }}{% else %}{{ workExperienceDetails.type | formatJobType }}{% endif %} experience details</span>
+                    </a>
+                  </dd>
+                </div>
+              {% endfor %}
+            {% endif %}
+            {% endif %}
+          </dl>
+          <p class="govuk-hint">Last updated: {{ workAndInterests.data.workExperience.updatedAt | formatDate('DD MMMM YYYY') }} by {{ workAndInterests.data.workExperience.updatedBy }}</p>
         </div>
       </div>
 
+      {% if workAndInterests.data.workInterests %}
```

Comment:
Looks like this can be empty, so added a conditional.

See data from `G0463UX` & `G0229UO`